<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>网元设置</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButtonOK">
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>网元信息</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelNE">
        <property name="text">
         <string>名称：</string>
        </property>
        <property name="buddy">
         <cstring>lineEditNE</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditNE"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelIP">
        <property name="text">
         <string>IP：</string>
        </property>
        <property name="buddy">
         <cstring>lineEditIP</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditIP"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>管理平台</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelManageUser">
        <property name="text">
         <string>用户：</string>
        </property>
        <property name="buddy">
         <cstring>lineEditManageUser</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEditManageUser"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelManagePassword">
        <property name="text">
         <string>密码：</string>
        </property>
        <property name="buddy">
         <cstring>lineEditManagePassword</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEditManagePassword">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>接入平台</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="labelAccessUser">
        <property name="text">
         <string>用户：</string>
        </property>
        <property name="buddy">
         <cstring>lineEditAccessUser</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineEditAccessUser"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="labelAccessPassword">
        <property name="text">
         <string>密码：</string>
        </property>
        <property name="buddy">
         <cstring>lineEditAccessPassword</cstring>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="lineEditAccessPassword">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>207</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
